recipes/masurca/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+echo `pwd`
+outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p $outdir
+mkdir -p $PREFIX/bin
+cd $SRC_DIR
+echo $SRC_DIR
+
+./install.sh
```

Comment:
You probably need to pass here the directory where you want to install your binaries ... $PREFIX/bin.